### PR TITLE
samples: fix lint and docs

### PR DESCRIPTION
**Pull request by jkwlui**

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "engines": {
+    "node": ">=8"
+  },
+  "repository": "googleapis/nodejs-talent",
+  "private": true,
+  "files": [
+    "*.js"
+  ],
+  "scripts": {
+    "cleanup": "node scripts/cleanup",
```
Comment (JustinBeckwith) [inline samples/package.json]: What's that do?

File: samples/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "test": "mocha system-test/*.js --timeout 600000"
```
Comment (JustinBeckwith) [inline samples/package.json]: I want to get away from `system-test` - can you just make this say `mocha` for now?

File: samples/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "dependencies": {
+    "@google-cloud/talent": "0.1.0",
+    "uuid": "^3.3.2",
```
Comment (JustinBeckwith) [inline samples/package.json]: Does this actually use any of these libs? Please remove if it doesn't.

**Comment by jkwlui** [comment]
@JustinBeckwith this will have to be forced in for now.. `docs-test` reporting dead link to the npm page for this package, which hasn't been published yet 🚫

File: synth.py [hunk 4]
```diff
@@ -46,6 +46,10 @@
         'toISOString)')
 # [END fix-dead-link]
```
[the remark this answers is not in the document]
Comment (jkwlui) [reply synth.py]: I think it's okay for links to be in quotes (`"`). The problem here is `jsdoc/markdown` incorrectly parses the link making it 404.

[the remark this answers is not in the document]
Comment (JustinBeckwith) [reply synth.py]: /me gently weeps

**Comment by jkwlui** [comment]
Please force merge :) @JustinBeckwith

**Pull request merged**